tf/workspace_tf2_4_0.bzl

```diff
@@ -0,0 +1,51 @@
+"""Install dependencies."""
```

Comment:
can you do git cp so that we can see diffs between different version?

Reply:
Sorry, I actually never used that and can't find anything apart from this [SO post](https://stackoverflow.com/questions/47401843/git-copy-file-as-opposed-to-git-mv). But I just ran this locally:
```bash
diff workspace_tf2_3_0.bzl workspace_tf2_4_0.bzl
```
with an empty output, so the files are indeed identical.